nullaway/src/main/java/com/uber/nullaway/GenericsChecks.java

```diff
@@ -498,4 +552,108 @@ public void compareGenericTypeParameterNullabilityForCall(
       }
     }
   }
+
+  public void checkTypeParameterNullnessForMethodOverriding(
+      MethodTree tree, Symbol.MethodSymbol overridingMethod, Symbol.MethodSymbol overriddenMethod) {
+    if (!config.isJSpecifyMode()) {
+      return;
+    }
+    checkTypeParameterNullnessForOverridingMethodReturnType(tree, overridingMethod);
+    checkTypeParameterNullnessForOverridingMethodParameterType(
+        tree, overridingMethod, overriddenMethod);
+  }
+
+  private void checkTypeParameterNullnessForOverridingMethodParameterType(
+      MethodTree tree, Symbol.MethodSymbol overridingMethod, Symbol.MethodSymbol overriddenMethod) {
+    List<? extends VariableTree> methodParameters = tree.getParameters();
+    List<Symbol.VarSymbol> overriddenMethodParams = overriddenMethod.getParameters();
+    Type ownerClassType =
+        state.getTypes().asSuper(overridingMethod.owner.type, overriddenMethod.owner);
+    if (ownerClassType == null) {
+      return;
+    }
+    for (int i = 0; i < methodParameters.size(); i++) {
+      Type methodParameterType = getTreeType(methodParameters.get(i));
+      // p
+      Type typeParameterTypeOverriddenMethod =
+          state.getTypes().memberType(ownerClassType, overriddenMethodParams.get(i));
```

Comment:
Rather than calling `memberType` for each overridden parameter type individually, why not call it once for the overridden method to get a `MethodType` and then get the parameter types from there?